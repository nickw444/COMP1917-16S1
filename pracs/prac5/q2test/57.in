57
-8 8 5 10 -4 -5 0 -2 -3 9 9 -10 -7 5 9 -6 -1 -7 -6 -8 -10 -3 4 6 9 3 0 -10 0 -9 2 8 -5 -3 -5 4 5 10 -3 7 -3 -1 -4 -4 -8 -5 4 3 9 -4 -8 -8 -3 9 6 9 8
