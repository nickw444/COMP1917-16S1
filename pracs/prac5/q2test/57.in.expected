average of array: -0.035088
